usr/lib/ep11_stdll/ep11.h

```diff
@@ -863,38 +727,46 @@ typedef enum {
 } XCP_LogSystem_t;
 
 /* bitmask of audit-event flags (mainly optional fields) */
-typedef enum {
-	XCP_LOGFL_WK_PRESENT         = 0x80000000,
-	XCP_LOGFL_COMPLIANCE_PRESENT = 0x40000000,  /* ...of hosting domain */
-	XCP_LOGFL_FINALWK_PRESENT    = 0x20000000,
-	XCP_LOGFL_KEYREC0_PRESENT    = 0x10000000,
-	XCP_LOGFL_KEYREC0_COMPL      = 0x08000000,  /* key0 compliance */
-	XCP_LOGFL_KEYREC1_PRESENT    = 0x04000000,
-	XCP_LOGFL_KEYREC2_PRESENT    = 0x02000000,
-	XCP_LOGFL_FINTIME_PRESENT    = 0x01000000,
-	XCP_LOGFL_SALT0_PRESENT      = 0x00800000,
-	XCP_LOGFL_SALT1_PRESENT      = 0x00400000,
-	XCP_LOGFL_SALT2_PRESENT      = 0x00200000,
-	XCP_LOGFL_REASON_PRESENT     = 0x00100000,
-	XCP_LOGFL_SEQPRF_PRESENT     = 0x00080000
-} XCP_LogFlags_t;
+#define  XCP_LOGFL_WK_PRESENT          0x80000000
+#define  XCP_LOGFL_COMPLIANCE_PRESENT  0x40000000  /* ...of hosting domain */
+#define  XCP_LOGFL_FINALWK_PRESENT     0x20000000
+#define  XCP_LOGFL_KEYREC0_PRESENT     0x10000000
+#define  XCP_LOGFL_KEYREC0_COMPL       0x08000000  /* key0 compliance */
+#define  XCP_LOGFL_KEYREC1_PRESENT     0x04000000
+#define  XCP_LOGFL_KEYREC2_PRESENT     0x02000000
+#define  XCP_LOGFL_FINTIME_PRESENT     0x01000000
+#define  XCP_LOGFL_SALT0_PRESENT       0x00800000
+#define  XCP_LOGFL_SALT1_PRESENT       0x00400000
+#define  XCP_LOGFL_SALT2_PRESENT       0x00200000
+#define  XCP_LOGFL_REASON_PRESENT      0x00100000
+#define  XCP_LOGFL_SEQPRF_PRESENT      0x00080000
 
 
 
 //---  importer PK types  ----------------------------------------------------
 typedef enum {
 	XCP_IMPRKEY_RSA_2048    = 0,
 	XCP_IMPRKEY_RSA_4096    = 1,
-	XCP_IMPRKEY_EC_P256     = 2,    /* EC, NIST P-256        */
-	XCP_IMPRKEY_EC_P521     = 3,    /* EC, NIST P-521        */
-	XCP_IMPRKEY_EC_BP256r   = 4,    /* EC, Brainpool BP-256r */
-	XCP_IMPRKEY_EC_BP320r   = 5,    /* EC, Brainpool BP-320r */
-	XCP_IMPRKEY_EC_BP512r   = 6,    /* EC, Brainpool BP-512r */
+	XCP_IMPRKEY_EC_P256     = 2,    /* EC, NIST P-256                     */
```

Comment:
Formatting? tabs?

Reply:
We take the ep11 header file(s) as is, no change (except the license header).